rust/rpm-test.yml

```diff
@@ -0,0 +1,47 @@
+# Maintained in https://github.com/coreos/repo-templates
+# Do not edit downstream.
+
+name: RPMs
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+permissions:
+  contents: read
+
+# don't waste job slots on superseded code
```

Comment:
Oh sweet, didn't know this magic incantation. This is a good example of the benefits of centralizing this stuff.